low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy

```diff
@@ -164,6 +164,22 @@ class SSPLDevDeploy:
             cmd = "yum install -y python3"
             _ = self._execute_cmd(cmd)
 
+            print("INFO: INSTALLING jsonschema==3.2.0")
+            cmd = "pip3 install jsonschema==3.2.0"
+            _ = self._execute_cmd(cmd)
+
+            print("INFO: INSTALLING requests==2.25.1")
+            cmd = "pip3 install requests==2.25.1"
+            _ = self._execute_cmd(cmd)
+
+            print("INFO: INSTALLING python-consul==1.1.0")
+            cmd = "pip3 install python-consul==1.1.0"
+            _ = self._execute_cmd(cmd)
+
+            print("INFO: INSTALLING paramiko==2.7.2")
+            cmd = "pip3 install paramiko==2.7.2"
+            _ = self._execute_cmd(cmd)
+
```

Comment:
"_" can be avoided. 
Just `self._execute_cmd(cmd)` would be fine instead of assigning to _.

Reply:
Agree. Done the changes.